components/automate-cs-oc-bifrost/habitat/config/sys.config

```diff
@@ -70,8 +70,9 @@
                        {file, "{{pkg.svc_var_path}}/logs/crash.log"},
                        {file_size, {{cfg.log.rotation_max_megabytes}}},  %% Size in MB
                        {files, {{cfg.log.rotation_max_files}}},
-                         {annotations, [requestor_id, created_authz_id, perf_stats, msg]}
-                       ]
+                       {perf_log, {{cfg.log.extended_perf_log}}},
```

Comment:
We do not need to add this config.
Modify this line 107:
`{enable_extended_perf_log, true},` to `{enable_extended_perf_log, {{cfg.log.extended_perf_log}}},`